### PR TITLE
Solution homework 

### PR DESCRIPTION
**Pull request by Pyatnitsa1990**
https://lms.ithillel.ua/groups/63c00b312482232c29370c19/homeworks/640f3306defa5f61a9ae10b4

### CODE REVIEW

File: src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Circle.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.hillel.piatnytsia.homeworks.homework4.part1;
+
+public class Circle implements SquareFigure {
+    private double r;
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Circle.java]: bad var name r, rename it

File: src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Square.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.hillel.piatnytsia.homeworks.homework4.part1;
+
+public class Square implements SquareFigure {
+    private double a;
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Square.java]: bad var name  a

File: src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Triangle.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package com.hillel.piatnytsia.homeworks.homework4.part1;
+
+public class Triangle implements SquareFigure {
+
+    private double a = 4;
+    private double h = 3;
+
```
Comment (shaitanaleksander) [inline src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Triangle.java:7]: same here never ever name vars like this  and  especially class level vars

**Review by shaitanaleksander — APPROVED** [review]

File: src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Triangle.java (hunk 3 above)
Comment (Pyatnitsa1990) [reply src/main/java/com/hillel/piatnytsia/homeworks/homework4/part1/Triangle.java:7]: ok, I understand.

**Pull request merged**